standalone/RL_plus_script.py

```diff
@@ -226,8 +244,8 @@ def test():
 
 
 def main():
-    # train()
-    test()
+    train()
+    # test()
```

Comment:
Please keep training commented out (we provide pretrained model to try out)

Reply:
Will do.